File: Tugas coding modul 5/coding py/PRAK504-2310817120005-Noviana_Nur_Aisyah.py
def reverse(x):
    rev = 0

    while x != 0:
        rev = rev * 10 + x % 10
        x //= 10 

    return rev

def main():
    A, B = map(int, input().split())
    A = reverse(A)
    B = reverse(B)
    C = A + B
    print(reverse(C))

main()

File: Tugas coding modul 5/coding c/PRAK504-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>
int reverse(int x){
    int rev = 0;

    while (x != 0) {
        rev = rev * 10 + x % 10;    //rumus untuk reverse
        x /= 10;
    }
    
return rev;
}

int main() {
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d",reverse(C));
}